src/mito/src/error.rs

```diff
@@ -56,6 +56,9 @@ pub enum Error {
         backtrace: Backtrace,
     },
 
+    #[snafu(display("Invalid primary key: {}", msg))]
+    InvalidPrimaryKey { msg: String, backtrace: Backtrace },
```

Comment:
This is also a same definition in datanode/error. Also there are some others(like `InvalidSql`) duplicates.

Reply:
The error in a different crate may have the same definition, it's acceptable.